### DISCUSSION TITLE
GitHub Profile

### DISCUSSION THREAD

**Discussion by amREDOX** [community/community#44489] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

Hi guys, after creating a repository with the same name as the GitHub username it is not showing as a special repository which I want to show as a GitHub profile. Can anyone please tell me how to enable the repository as a special repository (manually) so it can reflect in my profile?

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @amREDOX 

Your repo must be called amREDOX, a message will also appear saying that it is a special repo. The URL of this repo should look like this:

https://github.com/amREDOX/amREDOX

**Reply by amREDOX** [reply]
Thank you, bro. I have made a mistake in naming.
